includes/class-llms-blocks-assets.php

```diff
@@ -184,6 +193,19 @@ private function use_bc_assets() {
 		);
 	}
 
+	/**
+	 * Can a variation transform icon be an object.
+	 *
+	 * @since [version]
+	 *
+	 * @return boolean
+	 */
+	private static function can_variation_transform_icon_be_an_object() {
+		// https://github.com/gocodebox/lifterlms-blocks/issues/170.
```

Comment:
```suggestion
	/**
	 * Can a variation transform icon be an object.
	 *
	 * @since [version]
	 *
	 * @link https://github.com/gocodebox/lifterlms-blocks/issues/170
	 *
	 * @return boolean
	 */
	private static function can_variation_transform_icon_be_an_object() {
```